rts/Lua/LuaSyncedCtrl.cpp

```diff
@@ -6041,6 +6043,17 @@ static inline void ParseSmoothMeshParams(lua_State* L, const char* caller,
 }
 
 
+/***
+ * @function Spring.RebuildSmoothMesh
+ * @treturn nil
+ */
+int LuaSyncedCtrl::RebuildSmoothMesh(lua_State* L)
+{
+	smoothGround.MakeSmoothMesh();
```

Comment:
Would be nice to add four optional area parameters (x1, z1, x2, z2) to optimize many small updates on a big map.

Reply:
Making height map changes triggers updates in the smooth mesh anyway. This is for when the whole map has been changed and the smooth mesh needs to be refreshed faster than the 5-25 seconds it takes to process all the updates in the background.